config/default.js

```diff
@@ -3,6 +3,7 @@ var apiUrl = 'http://localhost:' + apiPort; // eslint-disable-line no-var
 var frontendUrl = 'http://localhost:8080'; // eslint-disable-line no-var
 
 module.exports = {
+    appName: 'New App',
```

Comment:
For now it is, but the app name can be used everywhere (If it can avoid the use of yeoman or something like that).
But I can change if you want